types/mimetext/index.d.ts

```diff
@@ -0,0 +1,100 @@
+// Type definitions for mimetext 2.0
+// Project: https://github.com/muratgozel/MIMEText
+// Definitions by: Josh Goldberg <https://github.com/JoshuaKGoldberg>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface MailLocationData {
+    addr: string;
+    name?: string;
+}
+
+export interface RecipientOptions {
+    type?: RecipientType;
+}
+
+export interface MailboxBoundaries {
+    alt: string;
+    mixed: string;
+}
+
+export interface MailboxInput extends MailLocationData, RecipientOptions {}
+
+export type InputType = 'OBJECT' | 'SPEC_COMPLIANT_TEXT' | 'TEXT';
+
+export type MIMEPlacement = 'content' | 'header';
+
+export type RecipientType = 'bcc' | 'cc' | 'to';
+
+export type TextFormat = 'text/html' | 'text/plain';
+
+export type MailLocation = string | MailLocationData;
+
+export type MessageHeaders = string[] | Record<string, string>;
+
+export interface MIMEHeader {
+    custom: boolean;
+    dump: (v: unknown) => unknown;
+    generator?: () => string;
+    name: string;
+    placement: MIMEPlacement;
+    value: string | null;
+}
+
+export interface EnvironmentContext {
+    toBase64: (input: unknown) => string;
+    store: MIMEHeader[];
+}
+
+export interface MailboxOptions {
+    type?: string;
+}
+
+declare class Mailbox {
+    constructor(input: string | MailboxInput, options?: MailboxOptions);
+    parseSpecCompliantText(text: string): MailLocationData;
+    createMailbox(): void;
+    findInputType(input: string): InputType;
+    getAddrDomain(): string;
+    dump(): string;
+    toObject(): Required<MailboxInput>;
+}
+
+declare class MIMEMessageContent {
+    constructor(data: string);
+    getHeaders(): Record<string, string>;
+    getHeader(key: string): string | undefined;
+    isAttachment(): boolean;
+    dump(envctx: Partial<EnvironmentContext>, boundaries: MailboxBoundaries): string;
+    setHeader(key: string, value: string): this;
+    setHeaders(headers: MessageHeaders): this;
+}
+
+declare class MimeMessage {
+    constructor(envctx: EnvironmentContext);
+    asAttachments(): MIMEMessageContent[];
+    asEncoded(): string;
+    asRaw(): string;
+    generateBoundaries(): MailboxBoundaries;
+    getHeader(key: string): string | undefined;
+    getMessageByType(type: string): MIMEMessageContent;
+    getRecipients(options?: RecipientOptions): Mailbox[];
+    getSender(): Mailbox;
+    getSubject(): string;
+    setAttachment(name: string, type: TextFormat, attachment: string): MIMEMessageContent;
+    setBcc(bcc: MailLocation | MailLocation[]): Mailbox[];
+    setCc(cc: MailLocation | MailLocation[]): Mailbox[];
+    setHeader(key: string, value: string): string;
+    setMessage(format: TextFormat, message: string, moreHeaders?: MessageHeaders): MIMEMessageContent;
+    setRecipient(recipient: MailLocation | MailLocation[], options?: RecipientOptions): Mailbox[];
+    setSender(sender: MailLocation | MailLocation[]): Mailbox;
+    setSubject(subject: string): void;
```

Comment:
```suggestion
    setSubject(subject: string): string;
```
https://github.com/muratgozel/MIMEText#setsubjectvalue
https://github.com/muratgozel/MIMEText/blob/25c534b11ed136f2518ae831015517a8b6d0a8c0/dist/browser/cjs/index.js#L580